docs/api/wanchain.md

```diff
@@ -1860,25 +1907,25 @@ When using dynamic order, you cannot send any of the assets to the zero address
 
 ### Gateway(provider, gatewayConfig)
 
-A `class` representing a smart contract on the Ethereum blockchain.
+A `class` representing a smart contract on the Wanchain blockchain.
 
 **Arguments**
 
 | Argument | Description
 |-|-
-| gatewayConfig.assetLedgerDeployOrderId | A `string` representing an Ethereum address of the [asset ledger deploy gateway](/#public-addresses).
-| gatewayConfig.actionsOrderId | A `string` representing an Ethereum address of the [actions order gateway](/#public-addresses).
-| gatewayConfig.valueLedgerDeployOrderId | A `string` representing an Ethereum address of the [value ledger deploy gateway](/#public-addresses).
-| provider | [required] An instance of an HTTP or MetaMask provider.
+| gatewayConfig.assetLedgerDeployOrderId | A `string` representing an Wanchain address of the [asset ledger deploy gateway](/#public-addresses).
+| gatewayConfig.actionsOrderId | A `string` representing an Wanchain address of the [actions order gateway](/#public-addresses).
```

Comment:
| gatewayConfig.actionsOrderId | A `string` representing a Wanchain address of the [actions order gateway](/#public-addresses).